Classes/EventListener/GetAuthorizationUrlSetLanguageEventListener.php

```diff
@@ -0,0 +1,34 @@
+<?php
+
+declare(strict_types=1);
+
+namespace Causal\Oidc\EventListener;
+
+use Causal\Oidc\Event\GetAuthorizationUrlEvent;
+use TYPO3\CMS\Core\Site\Entity\SiteLanguage;
+
+class GetAuthorizationUrlSetLanguageEventListener
+{
+    public function __invoke(GetAuthorizationUrlEvent $event): void
+    {
+        $settings = $event->settings;
+        $languageOption = $settings['oidcAuthorizeLanguageParameter'] ?? null;
+        if (empty($languageOption)) {
```

Comment:
no need for `empty()` here, the variable exists for sure.
`if ($languageOption)` should suffice.

Reply:
Because of the `?? null` in the previous line, `$languageOption` may be `null`. 
`if ($languageOption)` would be the opposite of what I need for the guard clause. I need to return if `$languageOption` is `null` or an empty string. 

`if (! $languageOption)` would evaluate to `true` if `$languageOption` contains a falsy string, such as `"0"`. Would you rather me use that?

Reply:
I forgot to type the negation. Sorry for that.
Yes "falsy" string is okay. It is the same behavior as `empty`  then.